core/GhcMod/Pretty.hs

```diff
@@ -47,8 +49,14 @@ renderSDoc sdoc = do
 
 gmComponentNameDoc :: ChComponentName -> Doc
 gmComponentNameDoc ChSetupHsName   = text $ "Setup.hs"
+#if MIN_VERSION_cabal_helper(0,8,0)
+gmComponentNameDoc ChLibName       = text $ "library"
+gmComponentNameDoc (ChSubLibName _)= text $ "library"
+gmComponentNameDoc (ChFLibName _)  = text $ "flibrary"
```

Comment:
These should include the name of the library in the output like the cases below.